pgpubsub/tests/test_core.py

```diff
@@ -106,9 +106,6 @@ def test_author_bulk_insert_notify(pg_connection):
     with atomic():
         authors = Author.objects.bulk_create(authors)
 
-    # TODO: Understand why pg_connection.poll() is only
```

Comment:
I believe this is the manifestation of the same issue namely it might happen that the notification is received during the commit of the transation and then without the explicit poll it is not visible.